pages/api/satistics.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs'
+import getSSRApolloClient from '../../api/ssrApollo'
+import { ME } from '../../fragments/users'
+import { CsvRequest, CsvRequestStatus } from '../../api/constants'
+import { gql } from '@apollo/client'
+
+export default async function handler (req, res) {
```

Comment:
Calling this endpoint without a session results in a 500 error:

command:

```
$ curl https://sn.ekzyis.com/satistics.csv
```

server logs:
```
app  | - error pages/api/satistics.js (10:24) @ id
app  | - error Error [TypeError]: Cannot read properties of null (reading 'id')
app  |     at handler (webpack-internal:///(api)/./pages/api/satistics.js:29:9)
app  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
app  |   digest: undefined
app  | }
app  |    8 |   const apollo = await getSSRApolloClient({ req, res })
app  |    9 |   console.log('satistics handler')
app  | > 10 |   const { data: { me: { id, csvRequest, csvRequestStatus } } } = await apollo.query({ query: ME })
app  |      |                        ^
app  |   11 |   console.log(id)
app  |   12 |   const fname = `satistics_${id}.csv`
app  |   13 |   res.setHeader('Content-Type', 'text/csv')
app  | satistics handler
app  | 14912
app  | - wait compiling...
app  | - event compiled successfully in 63 ms (425 modules)
app  | LND GRPC connection successful
app  | - error pages/api/satistics.js (9:24) @ id
app  | - error Error [TypeError]: Cannot read properties of null (reading 'id')
app  |     at handler (webpack-internal:///(api)/./pages/api/satistics.js:28:9)
app  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5) {
app  |   digest: undefined
app  | }
app  |    7 | export default async function handler (req, res) {
app  |    8 |   const apollo = await getSSRApolloClient({ req, res })
app  | >  9 |   const { data: { me: { id, csvRequest, csvRequestStatus } } } = await apollo.query({ query: ME })
app  |      |                        ^
app  |   10 |   const fname = `satistics_${id}.csv`
app  |   11 |   res.setHeader('Content-Type', 'text/csv')
app  |   12 |   res.setHeader('Content-Disposition', 'attachment; filename=satistics.csv')
```

Reply:
Thanks..fixed to gracefully fall into the 400 error case.